src/screens/Booking/components/Baggage/Baggage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import {Divider} from 'react-native-paper';
+import {t} from '@translations';
+import styles from './Baggage.styles';
+import BaggageItem from '../BaggageItem/BaggageItem';
+
+interface BaggageProps {
+  cabin: Boolean;
+  checked: Boolean;
+  setCabinValue: () => void;
+  setCheckedValue: () => void;
+}
```

Comment:
we need change this interface because it isn't right
```suggestion
  cabin: boolean;
  checked: boolean;
  setCabinValue(): void;
  setCheckedValue(): void;
}
```